src/lib/renderComponent.tsx

```diff
@@ -136,6 +152,10 @@ const renderComponent = <P extends {}>(
   return (
     <FelaTheme
       render={(theme: IThemePrepared) => {
+        if (_.isEmpty(theme) || !theme.provided) {
```

Comment:
lets focus on the case where there is some empty `theme` object provided. When this could happen and why we should filter for it? Currently we have default team being non-empty object (that automatically passes), so there is no need to check for it (`_.isEmpty` would be always `false`)

Reply:
removed `provided` flag